opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperForm.tsx

```diff
@@ -213,11 +224,19 @@ const CsvMapperForm: FunctionComponent<CsvMapperFormProps> = ({
                   <Typography>{t('Semicolon')}</Typography>
                 </div>
               </div>
-              <div className={classes.center} style={{ marginTop: 20 }}>
+              <div className={`${classes.center} ${classes.marginTop}`}>
```

Comment:
Same

Reply:
What do you mean ? Should I squash it to one class or there is already one known ?